corehq/apps/analytics/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import *
```

Comment:
another really minor: we try to avoid using `import *` and choose to be explicit about the exact imports